### PR TITLE
Linting all-in

### LINKED ISSUE #76: document eslint
@micahstubbs How are you running the ESLint added in #73 ?

I tried `./node_modules/.bin/eslint .` and saw the following:

```
Oops! Something went wrong! :(

ESLint couldn't find the plugin "eslint-plugin-jsx-a11y". This can happen for a couple different reasons:
```

Perhaps it would be nice to add a "lint" script to `package.json`, so as not to depend on a global `eslint` installation nor require developers to use `./node_modules/.bin` ?

### PR DESCRIPTION
**Pull request by curran**
Closes #76

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,22 +1,7 @@
...
-    "no-shadow": "off", 
-    "no-use-before-define": "off",
     "no-param-reassign": "off",
-    "no-alert": "off",
-    "no-nested-ternary": "off",
-    "no-return-assign": ["error", "always"],
-    "no-unused-vars": "off",
-    "newline-per-chained-call": "off",
-    "no-restricted-properties": "off"
+    "no-use-before-define": "off"
```
Comment (micahstubbs) [inline .eslintrc]: good to see the ignored rules updated.  these were from another project, with different concerns.  the fewer rules you ignore, generally speaking, the happier time you will have later 😄

**Comment by micahstubbs** [comment]
the improvements to how we use linting and to the pretest npm script look good to me 👍 

screen shot 2017-03-23 at 4 04 02 am

**Comment by micahstubbs** [comment]
for good measure, I also ran `npm run test` and manually verified that all the tests still pass 😄 

screen shot 2017-03-23 at 4 05 50 am

**Comment by curran** [comment]
Awesome. Thanks @micahstubbs !

**Pull request merged**